pkg/background/request_process.go

```diff
@@ -34,24 +38,38 @@ func (c *Controller) MarkUR(ur *urkyverno.UpdateRequest) (*urkyverno.UpdateReque
 		}
 		return ur, true, nil
 	}
-
 	handler = config.KyvernoPodName
 	ur.Status.Handler = handler
-	new, err := c.kyvernoClient.KyvernoV1beta1().UpdateRequests(config.KyvernoNamespace).UpdateStatus(context.TODO(), ur, metav1.UpdateOptions{})
-	return new, true, err
+	updateUR, err := c.PatchHandler(ur, string(handler))
```

Comment:
Another question: we mark the UR by the unique handler's name. With this change, all updates will be successful. This brings us a problem that multiple instances reconcile the same UR which is unexpected.

Reply:
Reverted the changes since we can use the patch here, replaced back to update